index.html

```diff
@@ -5860,107 +5860,71 @@ <h3>Vocabularies</h3>
+Implementations that depend on RDF vocabulary processing MUST ensure that the
+following vocabulary URLs used in the base context ultimately resolve to the
+following files, which are normative. Other semantically equivalent
+serializations of the vocabulary files MAY be utilized by implementations.
+Cryptographic hashes are provided for all content to ensure that developers can
+verify that the contents of each file are correct.
         </p>
 
         <table class="simple">
           <thead>
             <tr>
-              <th>URL</th>
-              <th>Media Type</th>
-              <th>Content</th>
+              <th>URL and Media Type</th>
+              <th>Content and Hashes</th>
             </tr>
           </thead>
           <tbody>
             <tr>
               <td>
-https://www.w3.org/2018/credentials#
-              </td>
-              <td>
-text/html
-              </td>
-              <td>
-https://www.w3.org/2018/credentials/index.html
-              </td>
-            </tr>
-            <tr>
-              <td>
-https://www.w3.org/2018/credentials#
-              </td>
-              <td>
+https://www.w3.org/2018/credentials#<br>
 application/ld+json
               </td>
               <td>
-https://www.w3.org/2018/credentials/index.jsonld
+https://www.w3.org/2018/credentials/index.jsonld<br><br>
+sha256: z52TgKqh2nqTCuACI8lCvhRdjwxQjeVmuOMCDCEijq4=<br><br>
+sha3-512: m8Ss+jgZiyL2Ws/ICJcWjHFd9PccJWsXPvMatBOhrH<wbr>
+h0qCBrzfgO2zO1OQQbTL7zoPgLseIbcxJJpunD2bkoRA==
               </td>
             </tr>
             <tr>
               <td>
-https://schema.org/
-              </td>
-              <td>
-text/html
-              </td>
-              <td>
-https://schema.org/
-              </td>
-            </tr>
-            <tr>
-              <td>
-https://schema.org/
-              </td>
-              <td>
+https://schema.org/<br>
 application/ld+json
               </td>
               <td>
-https://schema.org/version/latest/schemaorg-current-https.jsonld
-              </td>
-            </tr>
-            <tr>
-              <td>
-https://w3id.org/security#
-              </td>
-              <td>
-text/html
-              </td>
-              <td>
-https://w3c.github.io/vc-data-integrity/vocab/security/vocabulary.html
+https://schema.org/version/22.0/schemaorg-all-https.jsonld<br><br>
+sha256: KToiFier2I43VBuzvTkHG1dnBZEnpwoThC5AozLhnzA=<br><br>
+sha3-512: uao4ahYBx+0pl5os+tg1CaljyXxVN4CpHtC19tESQF<wbr>
+QYx7Rd5nQww/F8/ERpnsay1gDECReEyZzmffwhWV3omQ==
               </td>
             </tr>
             <tr>
               <td>
-https://w3id.org/security#
-              </td>
-              <td>
+https://w3id.org/security#<br>
 application/ld+json
               </td>
               <td>
-https://w3c.github.io/vc-data-integrity/vocab/security/vocabulary.jsonld
+https://w3c.github.io/vc-data-integrity/vocab/security/vocabulary.jsonld<br><br>
+sha256: LEaoTyf796eTaSlYWjfPe3Yb+poCW9TjWYTbFDmC0tc=<br><br>
```

Comment:
These don't seem to be correct right now.  If they are in flux, perhaps put in a temp FIXME value?

Reply:
They will be in flux for the forseeable future... I mentioned using FIXME values during one of the calls and it wasn't a popular option. Ideally, someone would write some code that retrieves these files and does the hash within the spec, but until that happens, this is going to be a manual process and perpetually broken until we get into CR.